src/storyHelpers/FontTokenSwatchGroup.svelte

```diff
@@ -0,0 +1,107 @@
+<script>
+  import Label from '../components/label/label.svelte'
+  import Icon from '../components/icon/icon.svelte'
+
+  export let level = 1
+  export let name = ''
+  export let tokens
+
+  const headingTag = `h${level}`
+
+  const getTokens = (obj) =>
+    Object.entries(obj).filter((i) => i[0] !== 'toString')
+
+  const handleClick = async ({ currentTarget }) => {
+    await navigator.clipboard.writeText(currentTarget.title)
+    currentTarget.classList.add('isCopied')
+    setTimeout(() => {
+      currentTarget.classList.remove('isCopied')
+    }, 1000)
+  }
+
+  const filteredTokens = getTokens(tokens)
+</script>
+
+<svelte:element this={name ? 'section' : 'div'} id={name}>
+  {#if name}
+    <svelte:element this={headingTag} class="group-heading"
+      >{name}</svelte:element
+    >
+  {/if}
+
+  {#if typeof filteredTokens[0][1] === 'string'}
+    <div class="font-group">
+      {#each filteredTokens as [name, token]}
```

Comment:
I'm not sure that we'd want to filter them out since they're part of the DS, and the idea is that Storybook should be the dev's source of truth for what's in the system.

Reply:
It just feels not very source of truthy if they don't display properly :D But yeah, probably fine